package.json

```diff
@@ -2,15 +2,32 @@
   "name": "cypher-validator",
```

Comment:
Not sure about the name since it will be used at NPM and maybe we don't want to steal that name. What do you think should it be updated?

Reply:
Steal the name from who? Neo4j can publish it under their @neo4j handle. I think the name is pretty good, but I can also put it under the @yworks org if you think that's better.

Reply:
I don't mind,  just wanted your input 😄